### PR TITLE
code-of-conduct: initial commit

### PR DESCRIPTION
**Pull request by philips**
And a code of conduct based on the well adopted Contributor Covenant
which is used in other large projects that CoreOS depends on such as
AngularJS, Kubernetes, Jekyll, curl, and many many more.

Also add the events code of conduct to cover any meetups, conferences,
or other social events based on the existing CoreOS Fest code of
conduct: https://coreos.com/fest/conduct/

### CODE REVIEW

**Comment by crawford** [comment]
The events section doesn't seem appropriate here.

**Comment by philips** [comment]
@crawford Why does it not seem appropriate? I think it is a good idea for no other reason than a reminder. We also have adhoc events like meetups around these projects that don't have landing pages, etc.

**Comment by crawford** [comment]
Because the events CoC has very little to do with contributing to the project. If people are attending the event, sure, include it. But asking people to email you and Mega seems out of place. I would just trim that section and include it in our meetup page and website.

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+should not engage in discriminatory or offensive speech or actions regarding
+gender, sexuality, race, age, disability, or religion. Speakers should be
+especially aware of these concerns.
+
+CoreOS does not condone any statements by speakers contrary to these standards.
+CoreOS reserves the right to deny entrance and/or eject from an event (without
+refund) any individual found to be engaging in discriminatory or offensive
+speech or actions.
+
+Please bring any concerns to to the immediate attention of designated on-site
```
Comment (xaka) [inline code-of-conduct.md]: double `to to` 🚂

**Pull request merged**